src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle() {
+    }
+
+    public Circle(int radius, Color color) {
+        this.radius = radius;
+    }
```

Comment:
your constructor accepts 2 arguments but doing nothing with color
```suggestion
    public Circle(int radius, String color) {
        this.radius = radius;
        this.color = color;
    }
```